.github/workflows/matrix.js

```diff
@@ -200,6 +200,7 @@ matrix.exclude(row => row.ssl.value === 'yes' && isLessThan(row.pg_version, '9.3
 matrix.exclude(row => row.scram.value === 'yes' && isLessThan(row.pg_version, '10'));
 matrix.exclude(row => row.replication.value === 'yes' && isLessThan(row.pg_version, '9.6'));
 
+matrix.exclude({query_mode: {value: 'simple'}, pg_version: '8.4'});
```

Comment:
It would be nice to have a comment linking to the failing case or something like that